### DISCUSSION TITLE
Anyone no longer able to merge lora model with their base model since the update ?

### DISCUSSION THREAD

**Discussion by agentj01** [d8ahazard/sd_dreambooth_extension#662] [Q&A]
Anyone no longer able to merge lora model with their base model since the recent update of the extension ?
I had to delete the extension and reinstall it because there was a bug on the automatic1111 webui where prompt result were not displaying (because of a gradio update it seems)
Even if a lora model is selected the output model seems to just be the base model.
The merging was working fine before the update.

**Comment by Boldor83** [comment]
yes can't use it either. i think there is a problem with the lora implemention now, it also uses way too much vram

**Reply by agentj01** [reply]
ho just noticed lora training just doesnt work too, i suppose we will have to chill until @d8ahazard fix it

**Reply by andykaufseo** [reply]
not sure what's going on, but mine worked fine, both training and merging, and i tested all previous commits
